packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -11,6 +11,16 @@ export 'package:flutter/foundation.dart' show debugPrint;
 // Any changes to this file should be reflected in the debugAssertAllRenderVarsUnset()
 // function below.
 
+/// Used by [debugDumpSemanticsTree] to specify the order in which child nodes
+/// are printed.
+enum DebugSemanticsDumpOrder {
+  /// Print nodes in inverse hit test order.
+  inverseHitTest,
+
+  /// Print nodes in traversal order.
+  traversal,
+}
```

Comment:
the descriptions of the two enum keywords are not very detailed. They should explain what they mean to someone who wasn't able to guess based on the keyword name, because if they had been able to guess from the keyword name, they wouldn't be reading the documentation.